### PR TITLE
iOS walkthrough tweaks

### PR DESCRIPTION
**Pull request by karimnaaji**
Minor changes mentioned in https://github.com/tangrams/tangram-docs/pull/209

### CODE REVIEW

File: pages/iOS-walkthrough.md [hunk 1]
```diff
@@ -41,10 +41,12 @@ pod install
 
 You can know more about setting up a Pod file from [the CocoaPods documentation](https://guides.cocoapods.org/using/the-podfile.html).
 
-If you are not using CocoaPods, you can download the latest version of the framework from [our releases](https://github.com/tangrams/tangram-es/releases), then simply drag and drop the Framework into your iOS project. Then, from your poject settings under `Embedded Binaries`, click on `+` to add `TangramMap.framework`, and make sure that the framework appears in the build phases with _Code Sign On Copy_ is checked.
+If you are not using CocoaPods, you can download the latest version of the framework from [our releases](https://github.com/tangrams/tangram-es/releases), then simply drag and drop the Framework into your iOS project. Then, from your poject settings under `Embedded Binaries`, click on `+` to add `TangramMap.framework`, and make sure that the framework appears in the build phases with _Code Sign On Copy_ checked.
 
 ![](images/XCode-Codesign.png)
 
+>If you are using CocoaPods, make sure to open your `xcworkspace` project (not `xcodeproj`).
```
Comment (tallytalwar) [inline pages/iOS-walkthrough.md:48]: If this is documented somewhere in cocoapod documentation, we should reference that here.

File: pages/iOS-walkthrough.md [hunk 2]
```diff
@@ -97,4 +99,4 @@ In this guide, you learned how to add a Tangram map to your iOS application. Now
 
 - Learn how to write or edit a Tangram [scene file](https://mapzen.com/documentation/tangram/Scene-file/),
 - See [more examples](https://github.com/tangrams/tangram-android-demos) of things your application can do with the map, or
-- Look at the [reference documentation](https://mapzen.com/documentation/tangram/ios-framework/0.6.0/) for the Tangram iOS framework.
\ No newline at end of file
+- Look at the [reference documentation](https://mapzen.com/documentation/tangram/ios-framework/0.6.0/) for the Tangram iOS framework.
```
Comment (tallytalwar) [inline pages/iOS-walkthrough.md]: 0.6.1

File: pages/iOS-walkthrough.md [hunk 3]
```diff
@@ -85,16 +87,15 @@ class ViewController: TGMapViewController {
...
 
 ### Next steps
 In this guide, you learned how to add a Tangram map to your iOS application. Now you can:
 
-- Learn how to write or edit a Tangram [scene file](https://mapzen.com/documentation/tangram/Scene-file/),
-- See [more examples](https://github.com/tangrams/tangram-android-demos) of things your application can do with the map, or
-- Look at the [reference documentation](https://mapzen.com/documentation/tangram/ios-framework/0.6.0/) for the Tangram iOS framework.
\ No newline at end of file
+- Learn how to write or edit a Tangram [scene file](https://mapzen.com/documentation/tangram/Scene-file/) or,
+- Look at the [reference documentation](https://mapzen.com/documentation/tangram/ios-framework/0.6.0/) for the Tangram iOS framework.
```
Comment (tallytalwar) [inline pages/iOS-walkthrough.md]: 0.6.1

File: pages/iOS-walkthrough.md [hunk 4]
```diff
@@ -41,10 +41,12 @@ pod install
 
 You can know more about setting up a Pod file from [the CocoaPods documentation](https://guides.cocoapods.org/using/the-podfile.html).
 
-If you are not using CocoaPods, you can download the latest version of the framework from [our releases](https://github.com/tangrams/tangram-es/releases), then simply drag and drop the Framework into your iOS project. Then, from your poject settings under `Embedded Binaries`, click on `+` to add `TangramMap.framework`, and make sure that the framework appears in the build phases with _Code Sign On Copy_ is checked.
+If you are not using CocoaPods, you can download the latest version of the framework from [our releases](https://github.com/tangrams/tangram-es/releases), then simply drag and drop the Framework into your iOS project. Then, from your poject settings under `Embedded Binaries`, click on `+` to add `TangramMap.framework`, and make sure that the framework appears in the build phases with _Code Sign On Copy_ checked.
```
Comment (tallytalwar) [inline pages/iOS-walkthrough.md:44]: Above update `pod 'Tangram-es', '~> 0.6.1'`

File: pages/iOS-walkthrough.md (hunk 1 above)
Comment (karimnaaji) [reply pages/iOS-walkthrough.md:48]: I don't think I've ever seen that in there, but I'll double check

**Review by tallytalwar — APPROVED** [review]

**Pull request merged**